server/deskroom/images/endpoints.py

```diff
@@ -103,3 +106,49 @@ async def upload_image(
     )
 
     return update_image_response.data[0]
+
+
+@router.delete("/{imageId}", status_code=status.HTTP_204_NO_CONTENT)
+async def delete_image(
+    image_id: str = Path(alias="imageId"),  # type: ignore
+    azure_container_client: ContainerClient = Depends(
+        create_source_azure_container_client
+    ),
+    supabase: AsyncClient = Depends(create_supabase_async_client),
+    current_user: DeskroomUser = Depends(get_current_user),
+) -> None:
+    image_response = (
+        await supabase.table("knowledge_images")
+        .select("*")
+        .eq("id", image_id)
+        .execute()
+    )
+    if not image_response.data:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="해당 이미지가 존재하지 않습니다.",
+        )
+    image = image_response.data[0]
+    if image.get("status") == "DELETED":
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="이미 삭제된 이미지입니다.",
+        )
+
+    image_url = image.get("image_url")
+    if image_url:
+        image_path = image_url.replace(f"{settings.AZURE_ACCOUNT_URL}/", "")
+        image_path = image_path.replace(f"{SOURCE_CONTAINER_NAME}/", "")
```

Comment:
```suggestion
        image_path = image_url.split(f"{settings.AZURE_ACCOUNT_URL}/{SOURCE_CONTAINER_NAME}/")[1]
```
요렇게 one-liner로도 가능하지 않을까요? 같은 변수로 해서 멱등하지 못할 거 같아보여서 줄여봤습니다.